Content.Shared/Stains/SharedStainsSystem.cs

```diff
@@ -0,0 +1,13 @@
+using Content.Shared.DoAfter;
+using Robust.Shared.Serialization;
+
+namespace Content.Shared.Stains;
+
+public abstract class SharedStainsSystem : EntitySystem
+{
+}
+
+[Serializable, NetSerializable]
+public sealed partial class SqueezeDoAfterEvent : SimpleDoAfterEvent
+{
+}
```

Comment:
Needs documenting, the event should say when it's raised / for what as it's very generic.